### ISSUE TITLE
demo?

### ISSUE THREAD

**Issue by dcsan** [aleofreddi/svgpan#40]
hey your demo link is down it seems :(

https://www.vleo.net/docs/projects/svgpan/tiger.svg

**Comment by aleofreddi** [comment]
Thank you for reporting this! I'll get it fixed!

**Comment by kfilippenok** [comment]
The demo does not work now.

**Comment by aleofreddi** [comment]
Yep, sorry about that, it's still broken. I'm moving my website around, I'll get it cleaned soon.
